examples/federation-workshop/cluster-2/safe/Deployment.yaml

```diff
@@ -0,0 +1,134 @@
+# /*
+# |    Protect your secrets, protect your sensitive data.
+# :    Explore VMware Secrets Manager docs at https://vsecm.com/
+# </
+# <>/  keep your secrets… secret
+# >/
+# <>/' Copyright 2023–present VMware, Inc.
+# >/'  SPDX-License-Identifier: BSD-2-Clause
+# */
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: vsecm-safe
+  namespace: vsecm-system
+  labels:
+    app.kubernetes.io/name: vsecm-safe
+    app.kubernetes.io/part-of: vsecm-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: vsecm-safe
+      app.kubernetes.io/part-of: vsecm-system
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: vsecm-safe
+        app.kubernetes.io/part-of: vsecm-system
+    spec:
+      serviceAccountName: vsecm-safe
+      containers:
+        - name: main
+          image: localhost:32000/vsecm-ist-safe:0.21.2
+          ports:
+            - containerPort: 8443
+          volumeMounts:
+            - name: spire-agent-socket
+              mountPath: /spire-agent-socket
+              readOnly: true
+            - name: vsecm-data
+              mountPath: /data
+            - name: vsecm-age
+              mountPath: /key
+              readOnly: true
+          #
+          # You can configure VSecM Safe by providing
+          # environment variables.
+          #
+          # See https://vsecm.com/configuration for more information about
+          # these environment variables.
+          #
+          # When you don’t explicitly provide env vars here, VMware Secrets Manager
+          # Safe will assume the default values outlined in the given link above.
+          #
+          env:
+            - name: SPIFFE_ENDPOINT_SOCKET
+              value: "unix:///spire-agent-socket/agent.sock"
+            - name: VSECM_LOG_LEVEL
+              value: "7"
+            - name: VSECM_WORKLOAD_SVID_PREFIX
```

Comment:
we’ll need more flexibility around these prefixes.

will create issues for that.